### PR TITLE
Add cloud dashboard command

### PR DESCRIPTION
**Pull request by gabrieljackson**
The cloud dashboard allows for monitoring all important cloud
resources via auto-refreshing CLI output.

This change also enhances group status functionality by providing
a count for the number of installations currently being updated.

Fixes https://mattermost.atlassian.net/browse/MM-29580

```release-note
Add cloud dashboard command
```

### CODE REVIEW

File: cmd/cloud/dashboard.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		}
+
+		writer := uilive.New()
+		writer.Start()
+
+		for {
+			tableString := &strings.Builder{}
+			table := tablewriter.NewWriter(tableString)
+			table.SetAlignment(tablewriter.ALIGN_LEFT)
+			table.SetHeader([]string{"TYPE", "TOTAL", "STABLE", "WIP"})
```
Comment (Szymongib) [inline cmd/cloud/dashboard.go:47]: Maybe something useful for the future would be to split `WIP` column into those that are actually being processed and those that are in a failed state, that are waiting for some action.

That is just an idea for the future, not something for this PR.

File: cmd/cloud/dashboard.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+			for i := refreshSeconds; i > 0; i-- {
+				fmt.Fprintf(writer, "%s\nUpdating in %d seconds...\n", renderedDashboard, i)
+				time.Sleep(time.Second)
+			}
+		}
+	},
+}
+
+func toStr(i int) string {
+	return fmt.Sprintf("%d", i)
```
Comment (Szymongib) [inline cmd/cloud/dashboard.go:151]: I believe that a more canonical way of doing it is using `Itoa`
```suggestion
	return strconv.Itoa(i)
```

File: model/group_status.go [hunk 3]
```diff
@@ -13,6 +13,7 @@ import (
 type GroupStatus struct {
 	InstallationsTotal          int64
 	InstallationsUpdated        int64
+	InstallationsBeingUpdated   int64
```
Comment (Szymongib) [inline model/group_status.go:16]: I like the idea of adding new field but I am not sure about the name, as by subtracting `updatedCount` and `awaitingUpdateCount` from `totalCount` we get all unstable Installations including those in failed states, hibernated or those being created/deleted.

I am afraid that is something that may prove a little confusing in the future. Maybe something like `InstallationsUnstable` would describe it more accurately?

**Review by Szymongib — APPROVED** [review]
Very cool feature 👍 
Just one small concern about naming 😉

File: model/group_status.go (hunk 3 above)
Comment (gabrieljackson) [reply model/group_status.go:16]: Yep, that's a good point. I will change the name here.

File: cmd/cloud/dashboard.go (hunk 1 above)
Comment (gabrieljackson) [reply cmd/cloud/dashboard.go:47]: Yeah, I like this idea for a future PR. Any resources that would fall in the `WIP` column will be listed in detail under the dashboard table so we won't miss that they may have failed, but having this in the table would also be nice.

**Review by stylianosrigas — APPROVED** [review]
Awesome 👌

**Pull request merged**